### ISSUE TITLE
UTF-8 characters cannot be deserialized

### ISSUE THREAD

**Issue by fluescher** [OpenHFT/Chronicle-Wire#193]
Hi,

we use Chronicle Queue and stumbled upon an issue when serializing/deserializing strings. 

We use Chronicle-Bytes indirectly but I tried to create a simple test:

```java
@Test
void readWriteString() {
    String utfCharacter = "ä";
        
    Bytes<ByteBuffer> bytes = Bytes.elasticByteBuffer();
    Wire wire = new BinaryWire(bytes);
    wire.getValueOut().writeString(utfCharacter);
        
    assertEquals(utfCharacter, wire.getValueIn().readString());
}
```
Which fails:

```
net.openhft.chronicle.bytes.UTFDataFormatRuntimeException: 1
00000000 e1 c3 a4                                         ···              


	at net.openhft.chronicle.bytes.BytesInternal.parseUtf8_SB1(BytesInternal.java:415)
	at net.openhft.chronicle.bytes.BytesInternal.parseUtf8(BytesInternal.java:138)
	at net.openhft.chronicle.bytes.StreamingDataInput.parseUtf8(StreamingDataInput.java:386)
	at net.openhft.chronicle.wire.BinaryWire.getStringBuilder(BinaryWire.java:721)
	at net.openhft.chronicle.wire.BinaryWire$BinaryValueIn.text(BinaryWire.java:2263)
	at net.openhft.chronicle.wire.ValueIn.readString(ValueIn.java:672)
	at UTFTest.readWriteString(UTFTest.java:22)
```

We use openjdk version "11.0.8" 2020-07-14 on Linux 5.7.17 x86_64. Maybe this is related to  https://github.com/OpenHFT/Chronicle-Queue/issues/553 ?

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.20.41](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.20.41), [BOM-2.20.67](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.67)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.20.101](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.20.101), [BOM-2.20.134](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.20.134)
